Lib/test/support/__init__.py

```diff
@@ -844,7 +844,9 @@ def dec(*args, **kwargs):
     '\u20AC',
 ):
     try:
-        os.fsdecode(os.fsencode(character))
+        if os.fsdecode(os.fsencode(character))\
```

Comment:
Why does this condition take two lines?

Reply:
I rebased it from 2.7 where the expression is longer. No prob, will reformat.